src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -219,7 +227,9 @@ public static String stringToHtml(String inxml, String infilexsl) {
 			transformer.transform(source, result);
 
 		} catch (TransformerException | SAXException | ParserConfigurationException | IOException e) {
+			showDialogForGUI();
 			logger.error(e.getMessage(), e);
+			return inxml;

```

Comment:
This one is not necessary, the next statement will return (without doing any I/O operation).